qiskit/pulse/commands/frame_change.py

```diff
@@ -41,3 +45,43 @@ def __eq__(self, other):
                 self.phase == other.phase:
             return True
         return False
+
+    def __repr__(self):
+        return '%s(%s, phase=%.3f)' % (self.__class__.__name__, self.name, self.phase)
+
+    def __call__(self, channel: OutputChannel) -> 'FrameChangePulse':
+        return FrameChangePulse(self, channel)
+
+
+class FrameChangePulse(Pulse):
```

Comment:
Framechange is also not a pulse, this is a `Schedule` at this point.

Reply:
I changed the name of the interface for "commands with channel context" from Pulse -> Instruction, e.g.
```
class FrameChangeInstruction(Instruction):
```
And `Schedule` has the same `Instruction` interface.
```
class Schedule(Instruction, ScheduleNode):
```
Can we push making `Instruction` be a `Schedule` to `functional-style` PR? 
(since I currently have no clear implementation image to do so.)
In this PR, I'd like to focus on making `Command` callable to show how UI around commands changes to internal users earlier.